### ISSUE TITLE
mdiswiz version must be updated for each mdiswiz release

### ISSUE THREAD

**Issue by dpfeuffer** [MEN-Mikro-Elektronik/13MD05-90#71]
The version of the mdiswiz consists of a common part and a OS specific part.
Example:
```
mdiswiz -h
mdiswiz 2.05.00-linux-13.0   Qt version 4.6.0
```
common part: 2.05.00 <-- specified in MDISWIZ\COM\mdiswiz.cpp
OS part: linux-13.0 <-- specified in MDISWIZ\COM\oslinux.cpp

The common part of the version is specified as:
  Common version x.yy.zzz
  x.yy = Major/Minor version number
  zzz   = Reflects changes in the CPU/BBIS model support (is not resetted on x.yy changes!)

Unfortunately, the zzz part was reset from 025 to 00 at:
Revision 2.16  2015/05/26 16:43:51  ts
So we have to fix this an set it to > 028

I have to maintain for Windows the MDISWIZ still at MEN in our old MCVS.
To be able to distinguish between the one in MCVS and the other at GitHub, please use 3 as major version for the GitHub mdiswiz.

**To sum it up, please:**
1. for the next release of mdiswiz:
  - modify in mdiswiz.cpp: mdiswizVersion = "3.00.030"
  - modify in oslinux.cpp: mdiswizLinuxVersion("linux-14.0")
2. observe the revision rule of the commen part as noted in mdiswiz.cpp for further releases
3. define a rule for the mdiswizLinuxVersion, note it in oslinux.cpp and observe it for further releases
4. Remark in the 13Y008-90 README.md that the mdiswiz version must be updated in the sources for each release!

**Comment by mlassota** [comment]
Fixed
